src/app/app.component.ts

```diff
@@ -21,13 +22,18 @@ export class SmiloWallet {
   selectedTheme: string;
 
   constructor(private platform: Platform, 
-              private statusBar: StatusBar, 
+              private androidPermissions: AndroidPermissions,
               private translate: TranslateService,
               private walletService: WalletService,
               private hockeyApp: HockeyApp,
-              private settingsService: SettingsService) {
+              private settingsService: SettingsService,
+              private statusBar: StatusBar) {
+    this.statusBar.styleLightContent();
     this.platform.ready().then(() => {
-      this.statusBar.styleDefault();
+
+      if (this.platform.is("android")) {
+        this.androidPermissions.requestPermissions([this.androidPermissions.PERMISSION.READ_EXTERNAL_STORAGE, this.androidPermissions.PERMISSION.WRITE_EXTERNAL_STORAGE]);
```

Comment:
Why don't you handle the promise result? Isn't it important to know if the user accepted?

Reply:
No, if the user didn't accept the store file function will simply fail with a message it could not be saved (toast message on bottom).